packages/fether-react/public/index.html

```diff
@@ -4,8 +4,25 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  <!-- CSP defined in packages/fether-electron/src/main/app/methods/setupRequestListeners.js -->
   <meta name="theme-color" content="#000000">
+    <!-- These CSP are for prod. For dev, CSP are set inside @electron-app -->
+    <meta http-equiv="Content-Security-Policy" content="
+        block-all-mixed-content;
+        child-src 'none';
+        connect-src http: ws:;
```

Comment:
While I'm comfortable putting http: for light-ui, here we're on mainnet with actual money, so just to be safe I'd remove http. Note: this will break hot-reloading in `yarn start`